SocialFilter/sections/specific.txt

```diff
@@ -186,9 +185,7 @@ tg-uchi.jp##div[class^="style_snsButtons"]
 censor.net##.block-share-link
 wccftech.com##.story-share-bottom
 opensubtitles.com##.social-share-button
-den3marta.felipevarela.com##.entry-social
 ||paparazzi.ru/banners/telegram.jpg
-newsweekjapan.jp###socialTool
```

Comment:
why removed?

Reply:
I think this rule is already add.

https://github.com/AdguardTeam/AdguardFilters/blob/21bc4b5391797424836da2050a2c48a4c5a81899/SocialFilter/sections/specific.txt#L3178